### PR TITLE
Add section on Challenge Solve Rate (CSR)

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses https://jira.cfops.it/browse/PCX-1021.

### CODE REVIEW

File: products/firewall/src/content/cf-firewall-rules/index.md [hunk 1]
```diff
@@ -28,19 +28,36 @@ You can also manage Firewall Rules through Terraform. For more, see [_Getting St
...
-![Firewall Rules tab](../images/firewall-rules-introduction-1.png)
+![Firewall Rules tab](../images/cf-firewall-rules-panel.png)
+
+#### Challenge Solve Rate (CSR)
+
+The **Rules List** displays each rule's **CSR** (Challenge Solve Rate), which is the percentage of solved CAPTCHA challenges considering the number of issued challenges. This metric applies to rules configured with _Challenge (Captcha)_ or _JS Challenge_ actions, and it is calculated as follows:
+
+<p><var>CSR</var> = <var>number of challenges solved</var> / <var>number of challenges issued</var></p>
+
+Hover the CSR value to reveal the number of issued and solved CAPTCHA challenges:
```
Comment (kodster28) [inline products/firewall/src/content/cf-firewall-rules/index.md]: Hover `over`?

File: products/firewall/src/content/cf-firewall-rules/index.md [hunk 2]
```diff
@@ -28,19 +28,36 @@ You can also manage Firewall Rules through Terraform. For more, see [_Getting St
...
+
+![Revealing the number of issued vs. solved CAPTCHA challenges](../images/firewall-rules-csr-hover.png)
+
+A low CSR value means that Cloudflare is issuing a low number of CAPTCHA challenges to actual humans, since these are the solved challenges.
+
+You should aim for a low Challenge Solve Rate. Review the CSR value of your CAPTCHA rules periodically and adjust them if necessary:
```
Comment (kodster28) [inline products/firewall/src/content/cf-firewall-rules/index.md:45]: The threshold for Bot Management is around `3%`. Not sure if you want to get that specific here, but I don't know if `low` is specific enough to be helpful to me as a user.

**Review by kodster28 — APPROVED** [review]
Two minor nits.

File: products/firewall/src/content/cf-firewall-rules/index.md (hunk 2 above)
Comment (pedrosousa) [reply products/firewall/src/content/cf-firewall-rules/index.md:45]: Would something like this work? The number may seem a bit out of the blue:
> You should aim for a low Challenge Solve Rate (up to 3%).

**Review by migueldemoura — APPROVED** [review]

**Pull request merged**